src/NBS.Appointments.Service.Core/Dtos/Qflow/Descriptors/QflowAppointmentDescriptor.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace NBS.Appointments.Service.Core.Dtos.Qflow.Descriptors
+{
+    public class QflowBookAppointmentDescriptor
+    {
+        public static QflowBookAppointmentDescriptor FromString(string descriptor)
+        {
+            var parts = descriptor.Split(":");
+
+            if (parts.Length != 7)
```

Comment:
I think this is fine for this scenario but in the future maybe let's look to abstract this logic in some more reusable way? Like if we're likely to have many of these, it'd be nice to have some declarative way to state what we expect